backend/functions/src/email/functions.ts

```diff
@@ -0,0 +1,138 @@
+import * as emails from './templates'
+import * as nodemailer from 'nodemailer'
+import { config } from 'dotenv'
+config()
+
+const transport = {
+  host: 'smtp.gmail.com',
+  port: 465,
+  secure: true,
+  auth: {
+    user: process.env.EMAIL_EMAIL,
+    pass: process.env.EMAIL_PASS,
+  },
+}
+
+const transporter = nodemailer.createTransport(transport)
+
+transporter.verify((err, succ) => {
+  if (err) {
+    console.log(' ❌ Transporter error: ❌ \n ', err)
+  } else {
+    console.log(
+      ' \n  ✅ Nodemailer transporter working ✅  \n Successful Message: ',
+      succ
+    )
+  }
+})
+
+const parseRecipients = (recipients: string[]) => {
+  let res = ''
+  recipients.forEach((e) => {
+    res += ', ' + e
+  })
+  return res
+}
+
+const sendEmail = (req: any, res: any) => {
+  const {
+    // name,
+    // email,
+    recipientsList,
+    message,
+    template,
+    newStudent,
+    existingStudents,
+    className,
+    customSubject,
+    customBody,
+  } = req.body
```

Comment:
In the current backend we usually destructure the request body in routes.js and pass the values to the function in functions.js. I think that's preferable because it's more clear what the parameters for `sendEmail` are, as opposed to `req: any` and having to look at the implementation to see what `req.body` should contain.

Also why are name and email commented out but present in the testing screenshot?